instrumentation/grape/lib/opentelemetry/instrumentation/grape/handler.rb

```diff
@@ -28,15 +26,68 @@ def subscribe
 
           private
 
-          def endpoint_start_process(name, start, finish, id, payload); end
+          def endpoint_run(name, start, finish, id, payload)
+            # TODO: check span.name and span.type
+            tracer.in_span(name, attributes: build_run_attributes(payload)) {}
```

Comment:
my concern with this approach is that the `in_span` method generates the span and marks the finish to the current timestamp when the method finishes executing. This is okay as it will be close enough to the current timestamp but it's possible that the finish timestamp might have a difference with the current timestamp that is significant. Perhaps something we should consider (might not need to deal with it now though 👍